lib/init.g

```diff
@@ -927,6 +927,17 @@ BindGlobal ("ShowSystemInformation", function ()
     ShowPackageInformation();
 end );
 
+
+#############################################################################
+##
+## Initialize the IsLIBGAP variable (if not done before). If this variable is false, an interactive
```

Comment:
Please line-wrap this (I think at 78 chars... anyway, at the same width as the `####...` line above.